MultipleImages/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import uuid from 'uuid';
+import styles from './style.css';
+import Image from '../Image';
+
+const getImageWrapperHeight = (arrayOfImageUrls) => {
+  if (arrayOfImageUrls.length <= 2) {
+    return { height: '100%' };
+  }
+};
```

Comment:
Hey, @alvarezmelissa87! I'm keen to grab your thoughts on this change. 😄 I've renamed this function and also its parameter to differentiate the actual argument we're passing in on line `29`. Curious to see how this feels with you? 😄